src/coreclr/vm/ceemain.cpp

```diff
@@ -1664,7 +1664,7 @@ static uint32_t g_flsIndex = FLS_OUT_OF_INDEXES;
 #define FLS_STATE_ARMED 1
 #define FLS_STATE_INVOKED 2
 
-static thread_local byte t_flsState;
+static __declspec(thread) byte t_flsState;
```

Comment:
This may want to stay as `_declspec(thread)` too since this is Windows-specific low-level hack. We do not want the C++ compiler to do something unexpected with it.